cns/api.go

```diff
@@ -18,6 +18,7 @@ const (
 	GetIPAddressUtilizationPath = "/network/ip/utilization"
 	GetUnhealthyIPAddressesPath = "/network/ipaddresses/unhealthy"
 	GetHealthReportPath         = "/network/health"
+	NumberOfCPUCoresPath        = "/hostcpucores"
```

Comment:
can it be compute/hostcpucores?

Reply:
let's not debate this to much. cns - container networking service.

Reply:
oki'm not forcing this. just a suggestion. cpucores ususally tie with compute.